.markdownlint.yaml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
Where did you pick this template? It should be https://github.com/dnsimple/dnsimple-engineering/blob/main/templates/markdownlint.yaml